### ISSUE TITLE
documentation as pdf (bundled)

### ISSUE THREAD

**Issue by 517qf** [renerocksai/sublimeless_zk#69]
Some people prefer a printout. This might slightly increase the chance that people read it and might avoid some questions in the future. It also feels more professional.

If I just run `pandoc -o README.pdf README.md` it looks quite well. There are only a hand full of overlong lines. My knowledge about latex is limited. My solution with inserting "\emergencystretch=10pt" into the template didn't help.

Maybe you can include this in your build-script?

**Comment by renerocksai** [comment]
Yes and no :smile:. In theory, a good idea. But this is just a little open-source project on GitHub, at the moment. I am trying to keep up with documentation and the best trade-off, at the moment, is maintaining the README. To be effective, I include animated GIFs for the non-readers. The animated GIFs usually don't turn out well in PDFs.

So, yes, something can be done and I know how to create good PDF / LaTeX from Markdown. However, I see the risk of having to maintain 2 docs then: the README and the PDF source.
